### PR TITLE
reactify

### PR DESCRIPTION
**Pull request by kzalite**

### CODE REVIEW

File: app/ts/Workspaces/WorkspacesTables/WorkspacesTable/WorkspacesTableRow/WorkspacesTableRow.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  index
+}: {
+  deleteLocalWorkspace: (id: string) => void;
+  workspace: IOldWorkspace;
+  index: number;
+}): JSX.Element {
+  const date = new Date(workspace.creationDate);
+  const datestring = dateFormat(date, 'yyyy-mm-dd');
+
+  function getLink(workspace: IOldWorkspace): string {
```
Comment (chinook25) [inline app/ts/Workspaces/WorkspacesTables/WorkspacesTable/WorkspacesTableRow/WorkspacesTableRow.tsx:20]: util?

File: app/ts/Workspaces/WorkspacesTables/WorkspacesTables.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      })
+      .catch(setError);
+
+    axios
+      .get('/api/v2/inProgress/')
+      .then((result: AxiosResponse<IInProgressWorkspaceProperties[]>) => {
+        setInProgressWorkspaces(_.sortBy(result.data, ['title']));
+      })
+      .catch(setError);
+  }, []);
```
Comment (chinook25) [inline app/ts/Workspaces/WorkspacesTables/WorkspacesTables.tsx:32]: so, the setters should be dependencies here I guess

**Review by chinook25 — APPROVED** [review]

File: app/ts/Workspaces/WorkspacesTables/WorkspacesTable/WorkspacesTable.tsx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        <TableRow>
+          <TableCell colSpan={type === 'finished' ? 4 : 2}>
+            <em id="empty-workspace-message">No workspaces defined</em>
+          </TableCell>
+        </TableRow>
+      </TableBody>
+    );
+  }
+
+  function orderByProperty(propertyToOrder: TSortProperty, event: any): void {
```
Comment (DanielReid) [inline app/ts/Workspaces/WorkspacesTables/WorkspacesTable/WorkspacesTable.tsx:96]: util?

**Review by DanielReid — APPROVED** [review]
Nice, some refactors. I pushed a start at getting eslint up & running

**Pull request merged**